java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -91,6 +94,13 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.configResetStat();
 
+        baseTransaction
+            .pfadd(hllKey1, new String[0])
```

Comment:
We should have just one test for transactions we can test functionality in shared commands. (This is consistent with Py and Node and Java)

Reply:
Reverted